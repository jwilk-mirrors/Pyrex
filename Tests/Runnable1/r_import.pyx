import spam
print "Imported spam"
d = dir(spam)
for n in d:
	if n[0] <> "_":
		print n

import sys
print "Imported sys"
print sys

