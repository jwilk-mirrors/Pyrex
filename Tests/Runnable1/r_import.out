Imported spam
['__builtins__', '__doc__', '__file__', '__name__', 'tons']
Imported sys
<module 'sys' (built-in)>
